### PR TITLE
Mark drag 'n' drop attributes as HTML-safe

### PR DESCRIPTION
**Pull request by jadefish**

### CODE REVIEW

**Comment by jadefish** [comment]
[sc-2085]

File: lib/coprl/presenters/web_client/helpers/drag_and_drop.rb [hunk 1]
```diff
@@ -1,21 +1,22 @@
 module Coprl::Presenters::WebClient::Helpers
   module DragAndDrop
     def draggable_attributes(comp)
-      draggable_attributes = ''
       if comp.draggable
-        draggable_attributes = "draggable=true data-drag_params='#{comp.draggable.to_h.to_json}'"
+        json = CGI.escapeHTML(comp.draggable.to_h.to_json)
+        html_safe("draggable=true data-drag_params=\"#{json}\"")
```
Comment (danbee) [inline lib/coprl/presenters/web_client/helpers/drag_and_drop.rb:6]: You can remove the `=true`.

**Review by danbee — APPROVED** [review]
LGTM

File: lib/coprl/presenters/web_client/helpers/drag_and_drop.rb (hunk 1 above)
Comment (jadefish) [reply lib/coprl/presenters/web_client/helpers/drag_and_drop.rb:6]: not quite. the component looks for `[draggable=true]`, which won't match element with a plain `draggable` attribute with no value.

```html
<!DOCTYPE html>
<html>
<body>
    <p id="no-value" something>no value</p>
    <p id="with-value" something="true">with value</p>
    <script>
        console.log(`document.querySelectorAll("[something=true]"):`, document.querySelectorAll("[something=true]"))
        console.log(`document.querySelectorAll("[something]"):`, document.querySelectorAll("[something]"))
    </script>
</body>
</html>
```

```
document.querySelectorAll("[something=true]"): NodeList [p#with-value]
document.querySelectorAll("[something]"): NodeList(2) [p#no-value, p#with-value]
```

**Pull request merged**